### ISSUE TITLE
[Discussion] Should SDKs surface common Metadata Keys?

### ISSUE THREAD

**Issue by berndverst** [dapr/dapr#3909]
Today the SDKs require that users specify metadata keys as freeform strings. It would be more convenient if SDKs provided common metadata key options natively.

@artursouza mentioned the Java SDK already provides common metadata keys, but this list is incomplete (needs link).

For example providing a TTL for State Store Set requests requires the metadata key `ttlInSeconds` to be set. SDKs could surface this and other metadata keys.

https://docs.dapr.io/developing-applications/building-blocks/state-management/state-store-ttl/
https://github.com/dapr/components-contrib/issues/306

**Comment by mukundansundar** [comment]
I agree providing common metadata keys in SDKs as constants would be better.
